index.html

```diff
@@ -0,0 +1,225 @@
+          <li><i class="bi bi-twitter"></i></li>
+          <li>English</li>
+          <li>My Page</li>
+          <li>Log out</li>
+        </ul>
+      </div>
+      <div class="container-fluid bg-white d-flex justify-content-center">
+        <div class="container d-flex justify-content-between p-4 fs-6 m-0">
+          <a href="index.html"
+            ><img src="img/AIBrain.ico" alt="AI brain logo" width="30px" /> AI
+            <span>B</span>rains</a
+          >
+          <ul class="d-flex fs-6 align-items-center m-0 gap-3 p-0">
+            <li><a href="about.html">About</a></li>
+            <li><a href="#">Program</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#">Sponsor</a></li>
+            <li><a href="#">News</a></li>
+            <li>
+              <a href="#" class="border border-4 border-danger p-2 text-danger"
+                >AI Campaign</a
+              >
+            </li>
+          </ul>
+        </div>
+      </div>
+    </header>
+    <!-- Main content -->
+    <main class="container-fluid p-0">
+      <!-- Intro section -->
+      <!-- Menu mobile -->
+      <div id="menu" class="hidden-mobile flex-column">
+        <div class="menu-mobile flex-column">
+          <button class="close-btn"><i class="bi bi-x-lg"></i></button>
+          <ul class="menu-options">
+            <li><a href="index.html">Home</a></li>
+            <li><a href="about.html">About</a></li>
+            <li><a href="#">Program</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#">Sponsor</a></li>
+            <li><a href="#">News</a></li>
+            <li>
+              <a href="#" class="border border-4 border-danger p-2 text-danger"
+                >AI Campaign</a
+              >
+            </li>
+          </ul>
+        </div>
+      </div>
+      <img
+        src="img/bg-intro-min.png"
+        alt="AI background image"
+        class="bg-image"
+      />
+      <section class="intro-container d-flex flex-column gap-3 p-3 p-lg-5">
+        <button class="hidden-mobile menu-btn btn align-self-start p-2">
+          <img src="img/btn-menu.png" alt="menu mobile" width="24px" />
+        </button>
+        <span class="fs-3 mt-5 title">"Hello! Human world"</span>
+        <h1 class="mb-3 title fw-bold">AI brains Global Summit 2023</h1>
+        <blockquote
+          class="col-sm-4 col-md-6 col-lg-6 border border-3 border-white fs-6 p-3"
+        >
+          "The best way to predict the future is to invent it." - Alan Kay
+        </blockquote>
+        <span class="fs-3 fw-bold subtitle">2023.04.22(SAT) ~ 23(SUN)</span>
+        <small class="fs-6">@The Artificial Minds Museum</small>
+      </section>
+      <!-- Main Program section -->
+      <section
+        class="main-program p-3 p-lg-5 d-flex flex-column align-items-center gap-3 gap-md-5"
+      >
+        <h2 class="text-white text-center">Main Program</h2>
+        <div class="guide-bar"></div>
```

Comment:
- [ ] Great job working on the desktop version so far but here I would suggest to decrease the padding/margin a little bit here to have it the same as the given design so kindly decrease it 🎉🚀

- Your
![image](https://user-images.githubusercontent.com/58553711/235039312-e84ac421-49f6-4d55-b7bf-016bb8855dfb.png)

- Given
![image](https://user-images.githubusercontent.com/58553711/235039358-9deb22b2-39ef-4ea6-932f-60b3f2faf96e.png)